### PR TITLE
compactified CQM equations

### PR DESCRIPTION
**Pull request by alexzucca90**
NFC

The two separate equality and inequality constraints in the `ConstrainedQuadraticModel` docstring are replaced with single more compact notation that also reflects the structure of the class.

### CODE REVIEW

File: releasenotes/notes/compactify-cqm-equations-5456497655b5e24d.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+features:
```
Comment (arcondello) [inline releasenotes/notes/compactify-cqm-equations-5456497655b5e24d.yaml:2]: I don't think this needs a release note.

**Review by JoelPasvolsky — COMMENTED** [review]
my one comment on the the implementation is whether `\circ` is the optimal symbol (is it standard? I'm not familiar with its use): it's small, so it looked at first glance a bit strange. 

I'm not sure I would find this easier to parse, it seems a trade between space (two equations) and time (one equation and then unpacking an unfamiliar symbol), but it works okay for me. 

We would want to also make [this](https://docs.ocean.dwavesys.com/en/stable/concepts/cqm.html) consistent.

**Comment by alexzucca90** [comment]
@JoelPasvolsky For the `\circ` symbol, that is used in some publications, e.g. equation 2.1 in [this publication](https://www.researchgate.net/publication/337127039_Presolve_Reductions_in_Mixed_Integer_Programming). For the Ocean docs, I can make an accompanying PR, maybe after merging this?

**Comment by JoelPasvolsky** [comment]
> @JoelPasvolsky For the `\circ` symbol, that is used in some publications, e.g. equation 2.1 in [this publication](https://www.researchgate.net/publication/337127039_Presolve_Reductions_in_Mixed_Integer_Programming). For the Ocean docs, I can make an accompanying PR, maybe after merging this?

Thanks, yes, I meant that the SDK update would be a different PR, it was just so we would keep it in mind.

**Comment by JoelPasvolsky** [comment]
@alexzucca90, I'm doing a little A/B testing and will get back to you soon.

**Comment by arcondello** [comment]
It might be slightly clearer to use `\bigcirc`, but have not tested it myself to be sure.

**Review by JoelPasvolsky — APPROVED** [review]

**Review by arcondello — APPROVED** [review]

**Pull request merged**